src/lib/utils/track.ts

```diff
@@ -3,6 +3,12 @@ import _track, { Track as _Track, TrackingInfo } from "react-tracking"
 
 import Events from "lib/NativeModules/Events"
 
+/**
+ * Useful notes:
+ *  * Add `jest.mock("react-tracking")` to test components
+ *    which are wrapped by tracking.
```

Comment:
Can’t this be done by default?

Reply:
Hrm, I think we could move that to the jest setup script and it wouldn't be required - I'm mid adding tests for the analytics calls - so I'll verify and flip the switch the other way, you unmock when needed